src/components/Slides.js

```diff
@@ -0,0 +1,118 @@
+                  <i className="fa fa-linkedin-square" />
+                </div>
+              </Card.Footer>
+            </Card>
+            <Card>
+              <Card.Img variant="top" src="./three.jpg" alt="Three" />
+              <Card.Body>
+                <Card.Title>Forest</Card.Title>
+                <Card.Text>
+                  In vulputate nulla sed dolor ultrices, a tincidunt lacus tincidunt.
+                </Card.Text>
+              </Card.Body>
+              <Card.Footer>
+                <div className="d-flex justify-content-center">
+                  <i className="fa fa-facebook-official" />
+                  <i className="fa fa-twitter" />
+                  <i className="fa fa-linkedin-square" />
+                </div>
+              </Card.Footer>
+            </Card>
+          </CardGroup>
+        </Carousel.Item>
+        <Carousel.Item interval={1500}>
+          <CardGroup>
+            <Card>
+              <Card.Img variant="top" src="./one.jpg" alt="Card  cap" />
+              <Card.Body>
+                <Card.Title>Desert Experience</Card.Title>
+                <Card.Text>
+                  Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+                </Card.Text>
+              </Card.Body>
+              <Card.Footer>
+                <div className="d-flex justify-content-center">
+                  <i className="fa fa-facebook-official" />
+                  <i className="fa fa-twitter" />
+                  <i className="fa fa-linkedin-square" />
+                </div>
+              </Card.Footer>
+            </Card>
+            <Card>
+              <Card.Img variant="top" src="./two.jpg" alt="TWO" />
+              <Card.Body>
+                <Card.Title>Beach</Card.Title>
+                <Card.Text>
+                  In vulputate nulla sed dolor ultrices, a tincidunt lacus tincidunt.
+                </Card.Text>
+              </Card.Body>
+              <Card.Footer>
+                <div className="d-flex justify-content-center">
+                  <i className="fa fa-facebook-official" />
+                  <i className="fa fa-twitter" />
+                  <i className="fa fa-linkedin-square" />
+                </div>
+              </Card.Footer>
+            </Card>
+            <Card>
+              <Card.Img variant="top" src="./three.jpg" alt="Three" />
+              <Card.Body>
+                <Card.Title>Forest</Card.Title>
+                <Card.Text>
+                  In vulputate nulla sed dolor ultrices, a tincidunt lacus tincidunt.
+                </Card.Text>
+              </Card.Body>
+              <Card.Footer>
+                <div className="d-flex justify-content-center">
+                  <i className="fa fa-facebook-official" />
+                  <i className="fa fa-twitter" />
+                  <i className="fa fa-linkedin-square" />
+                </div>
+              </Card.Footer>
+            </Card>
+          </CardGroup>
```

Comment:
[OPTIONAL]
Please consider populating these cards programmatically, so your code can stay DRY. You have classNames that repeat, as well as these tags inside your Card tags.